platform/core/src/services/ToolBarService/ToolbarService.ts

```diff
@@ -645,4 +734,56 @@ export default class ToolbarService extends PubSubService {
     this.state.buttonSections[buttonSection] = [];
     this._broadcastEvent(this.EVENTS.TOOL_BAR_MODIFIED, { ...this.state });
   }
+
+  /**
+   * Checks if a button exists in any toolbar section.
+   *
+   * @param buttonId - The button ID to check for
+   * @returns True if the button exists in any section, false otherwise
+   */
+  isInAnySection(buttonId: string): boolean {
```

Comment:
This is a fairly expensive operation, does it get used much?  If so then it probably needs a better implementation given how big/many sections are being added.

Reply:
Not really